xocto/storage/storage.py

```diff
@@ -808,10 +809,22 @@ def _get_policy(self) -> str | None:
         return None
 
     def _get_boto_client(self) -> S3Client:
+        config = {}
+
+        if settings.BOTO_S3_CONNECT_TIMEOUT:
+            config["connect_timeout"] = settings.BOTO_S3_CONNECT_TIMEOUT
```

Comment:
Seeing as this is an open-source package, I don't think we can be certain that these settings actually exist in Django's settings conf. Could we do something like this instead?


```suggestion
        if (connect_timeout := getattr(settings, "BOTO_S3_CONNECT_TIMEOUT")) is not None:
            config["connect_timeout"] = connect_timeout
```